moveit_setup_assistant/src/widgets/start_screen_widget.cpp

```diff
@@ -305,6 +305,13 @@ void StartScreenWidget::loadFilesClick()
   }
   else
   {
+    if ( !create_new_package_ && config_data_->author_name_.empty() && config_data_->author_email_.empty() )
+    {
+      QMessageBox::warning( this, "Loading Project Without Author Information",
+        "The loaded configuration package does not include author information.\n"
+        "Make sure you regenerate the '.setup_assistant' and 'package.xml' files to add it.");
```

Comment:
you're welcome to implement this :)
I will not be around for the next one-and-a-half weeks.
In any case, we have to tell the user to regenerate `.setup_assistant` because they are not able to generate *anything* if they don't have author information stored in there.

However, we could rephrase the notice.